apps/judicial-system/api/src/app/modules/case/models/case.model.ts

```diff
@@ -27,7 +27,6 @@ import {
 } from '@island.is/judicial-system/types'
 
 import { CivilClaimant, Defendant } from '../../defendant'
-import { EventLog } from '../../event-log'
```

Comment:
Event logs are no longer available to the frontend. Instead, select date fields have been added.